react.js

```diff
@@ -0,0 +1,71 @@
+module.exports = {
+	extends: ["airbnb", "airbnb/hooks"],
+	plugins: [],
+	env: {
+		es6: true,
+		node: true,
+		browser: true,
+	},
+	parserOptions: {
+		ecmaVersion: 6,
+		ecmaFeatures: {
+			jsx: true,
+		},
+		sourceType: "script",
```

Comment:
Are you sure about `script` here? I think it should be `module` since our source code is essentially modules.

Reply:
yeah you are right